app/jobs/subscription_placement_job.rb

```diff
@@ -87,4 +87,8 @@ def send_empty_email(order, changes)
     record_issue(:empty, order)
     SubscriptionMailer.empty_email(order, changes).deliver
   end
+
+  def logger
+    Delayed::Worker.logger
```

Comment:
it turns out we can't access the logger from a custom job :cry: :point_right: https://github.com/collectiveidea/delayed_job/blob/b78f88f11ed09b3c952406b08c2cc57e8e5b63bb/lib/delayed/backend/base.rb#L77-L90, so we can't use their formatting either :sob: 

What I'm considering is going with https://api.rubyonrails.org/classes/ActiveSupport/TaggedLogging.html in 
https://github.com/openfoodfoundation/openfoodnetwork/blob/74c7a01151d527e9f1f09fc5859e4faf66cb5a1e/config/initializers/delayed_job.rb#L1

so `Delayed::Worker.logger` gets much reacher formatting capabilities we can use ourselves.

Reply:
yeah, that sounds good to me!

Reply:
@mkllnk used tagged logging in the sample data:
https://github.com/openfoodfoundation/openfoodnetwork/blob/master/lib/tasks/sample_data/logging.rb